include/downloader.h

```diff
@@ -71,14 +71,13 @@ class Download {
    * - A first one to download the metadlink.
    * - A second one to download the real file.
    *
-   * By default, `Download` track only the first download. So status may appear
-   * as COMPLETE even if the real file downloading is still running.
-   * By passing true to follow, `Dowload` will detect that and will track the
-   * second download instead.
+   * By default, `Download` try to detect that and track the second download when
+   * the first one is finished.
+   * By passing false to follow, `Dowload` will only track the first download.
    *
    * `getFoo` methods are based on the last statusUpdate.
    */
-  void updateStatus(bool follow=false);
+  void updateStatus(bool follow=true);
```

Comment:
Since downloader is part of the libkiwix API, this semantic change can in theory lead to unpleasant surprises for clients of libkiwix who update to new version without reading the changelog carefully. A safer way would be introducing a new function with the new semantics and deprecating the old one.

Reply:
I agree with the problem you raise. But I would like to avoid adding a `updateStatus` for this.
Maybe we could  simply remove the default value. For people (other than us ?) using with `true` parameter it will change nothing.
For other using the default value (if there is one at all), it will be a compilation error.

Reply:
Yes, that's a better option